src/BuildingRegistry/Legacy/Building.cs

```diff
@@ -409,7 +409,7 @@ private static bool IsGeometryValid(ExtendedWkbGeometry validGeometry)
             }
 
             var geometry = WkbReader.Read(validGeometry);
-            return GeometryValidator.IsValid(geometry);
+            return GeometryValidator.IsValidMeasured(geometry);
```

Comment:
Don't change legacy.

Reply:
@ArneD changed it to IsValidOutlined. Will now fail on selftouching polygon.
or do we copy the GeometryValidator for legacy and leave it as it was?

Reply:
IsValidMeasured is same logic as before, so I think that's correct not outlined